products/compute/terraform.yaml

```diff
@@ -1360,6 +1360,24 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         exclude: true
       tunnels: !ruby/object:Overrides::Terraform::PropertyOverride
         exclude: true
+  VpnGateway: !ruby/object:Overrides::Terraform::ResourceOverride
+    name: 'HaVpnGateway'
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "ha_vpn_gateway_basic"
+        primary_resource_id: "ha_gateway"
+        vars:
+          ha_vpn_gateway_name: "ha-vpn-1"
+          network_name: "network1"
+    properties:
```

Comment:
I'd probably also exclude creationTimestamp (or just not include it in api.yaml)- I've never heard any Terraform user express any interest in that field on any resource.